### PR TITLE
Docs: add warning callout to add a workaround when jsDelivr is not available

### PR DESCRIPTION
**Pull request by julien-deramond**
### Description

We use (and advise to use) jsDelivr when using the CDN version of Bootstrap.

However, recently, in https://github.com/orgs/twbs/discussions/39143 or https://github.com/orgs/twbs/discussions/38782 (if I remember well, other discussions exist as well), it seems like jsDelivr is not available in some countries (might be blocked).

So this PR suggests adding a warning callout to explain it and provides a workaround with [cdnjs](https://cdnjs.com) pr [unpkg](https://unpkg.com/) which seem to be reliable resources as well.

As mentioned in the comments, [cdnjs Bootstrap page](https://cdnjs.com/libraries/bootstrap) provides a ready-to-use snippet that can be really a handful for folks.

### Type of changes

- [x] Enhancement (non-breaking change which adds functionality)

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] I have updated the documentation accordingly
- (N/A) I have added tests to cover my changes
- [x] All new and existing tests passed

#### Live previews

- https://deploy-preview-39145--twbs-bootstrap.netlify.app/docs/5.3/getting-started/download/#cdn-via-jsdelivr

### Related discussions

- https://github.com/orgs/twbs/discussions/39143
- https://github.com/orgs/twbs/discussions/38782

### CODE REVIEW

**Comment by iiAhmedYT** [comment]
As well as cdnjs (https://cdnjs.com/)
I'm in Egypt which is one of those countries where jsDelivr is blocked which is VERY annoying, i switched to cdnjs which is crazy fast, i love it

**Comment by julien-deramond** [comment]
Yep the 3 that I've often found are jsDelivr, unpkg, and cdnjs. I can maybe mention it as well if it can be useful for folks stuck with this issue.

**Comment by coliff** [comment]
One problem with this callout is that if a user were to do as it says and copy the code above

```
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-yvuw7Qka5HyIDWlQLof1Eh7TsntO8T9n5xVQ2fLEK8UEb47oC0LAx6N9KUadRsT8" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-AN46EzEu8aYMuLb9XLkZgyvOCua6xS1HKstfL9xcOyx+UKZYcp2Z58pZjezzj/Qc" crossorigin="anonymous"></script>
```
And replace `https://cdn.jsdelivr.net/npm/` with `https://unpkg.com/` then the links wouldn't work because the SRI appears to be different.
Quick test: https://stackblitz.com/edit/bootstrap-cdn-test?file=index.html

Maybe link to this page as an alternative CDN instead: https://cdnjs.com/libraries/bootstrap ?

**Comment by julien-deramond** [comment]
Yeah @coliff the integrity value must be changed to `<link rel="stylesheet" href="https://unpkg.com/bootstrap@5.3.1/dist/css/bootstrap.min.css" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">` for example for this file. Integrity value can be generated via https://www.srihash.org/ but it's not really user-friendly I suppose (and missing in my warning callout message).

Might be worth pointing to https://cdnjs.com/libraries/bootstrap instead as you said which offers a "Copy Script Tag".

Changing back this PR to draft mode and I'll do the modification later on.

**Comment by julien-deramond** [comment]
Based on your comments, I've refined this warning callout message. Don't hesitate to suggest some modifications if the content is not clear/helpful.

File: site/content/docs/5.3/getting-started/download.md [hunk 1]
```diff
@@ -50,6 +50,16 @@ If you're using our compiled JavaScript and prefer to include Popper separately,
...
+{{< callout warning >}}
+**Heads up!** We use and advise to use [jsDelivr](https://www.jsdelivr.com/) in our documentation. However, in some cases, like in some specific countries or environments, you might need to use other CDN providers, such as [cdnjs](https://cdnjs.com/) or [unpkg](https://unpkg.com/).
+
+You'll find the same files on these CDN providers, albeit with different URLs.
+
+While changing the URLs, you'll also need to update the `integrity` attribute. Tools like [SRI Hash Generator](https://www.srihash.org/) can help you generate the correct values.
+
+Easier, this [cdnjs Bootstrap direct link](https://cdnjs.com/libraries/bootstrap) provides a ready-to-use HTML snippet for each version of Bootstrap where you can just copy and paste the script tag.
+{{< /callout >}}
+
```
Comment (mdo) [inline site/content/docs/5.3/getting-started/download.md:62]:
```suggestion
### Alternative CDNs

We recommend [jsDelivr](https://www.jsdelivr.com/) and use it ourselves in our documentation. However, in some cases—like in some specific countries or environments—you may need to use other CDN providers like [cdnjs](https://cdnjs.com/) or [unpkg](https://unpkg.com/).

You'll find the same files on these CDN providers, albeit with different URLs. When changing the URLs, you'll also need to update the `integrity` attribute. Tools like [SRI Hash Generator](https://www.srihash.org/) can help you generate the correct values.

With cdnjs, you can [use this direct Bootstrap package link](https://cdnjs.com/libraries/bootstrap) to copy and paste ready-to-use HTML snippets for each dist file from any version of Bootstrap.

```

**Review by mdo — APPROVED** [review]
Let's not make this a callout. Added a suggested edit to add before merging.

**Pull request merged**